### ISSUE TITLE
Is `/content/index.mdx` being used? 

### ISSUE THREAD

**Issue by julieg18** [iterative/cml.dev#213]
As far as I can tell `content/index.mdx` isn't being used. Instead, the home page is formed from `src/pages/index.tsx`. Should we delete the page or is it being used somewhere that I missed? 

cc @rogermparent @yathomasi

**Comment by yathomasi** [comment]
> As far as I can tell `content/index.mdx` isn't being used. Instead, the home page is formed from `src/pages/index.tsx`. Should we delete the page or is it being used somewhere that I missed?

Yes, the home page is being generated from `src/pages/index.tsx`.  

Could it be still there because it would be helpful in maintaining the code? The HTML version seems messy.

<img width="1001" alt="Screen Shot 2022-03-25 at 19 29 04" src="https://user-images.githubusercontent.com/20840228/160132306-b8b90541-39dc-4238-88bd-4b3ef03940a6.png">

**Comment by julieg18** [comment]
> Could it be still there because it would be helpful in maintaining the code? 

Possibly! We also may have forgotten to delete it when we added docs to the site (#60).

> The HTML version seems messy.

Looking at both files, they both seem messy in their own way 😅

**Comment by rogermparent** [comment]
I wrote the original in MDX with intentions for the whole site to be MDX-based, this was back at peak Gatsby+MDX hype. I still see merits in an MDX-based approach, but considering the JSX version is the one we have now I'm pretty sure we can safely just drop the MDX version.

I believe @julieg18 is correct here in that the MDX file was simply forgotten when the docs implementation was added and for some reason a decision was made to change it to JSX, especially there was quite a few other leftover bits from the dvc.org copy+paste when it was done.
